DirectProgramming/DPC++FPGA/ReferenceDesigns/merge_sort/src/main.cpp

```diff
@@ -49,9 +49,30 @@ static_assert(fpga_tools::IsPow2(kMergeUnits));
 #define SORT_WIDTH 4
 #endif
 constexpr size_t kSortWidth = SORT_WIDTH;
-static_assert(kSortWidth >= 1);
+static_assert(kSortWidth >= 1); 
 static_assert(fpga_tools::IsPow2(kSortWidth));
 
+//
```

Comment:
I think that this is no longer needed (or is about to not be needed anymore) 
Here is Mark's PR for adding simulation targets: https://github.com/oneapi-src/oneAPI-samples/pull/1070
With the associated dpcpp change: https://github.com/intel/llvm/pull/6715
I'm going to ping him to know if his change is now integrated

Reply:
How can I use Mark's merged PR in my local repo to run the test with `ext::intel::fpga_simulator_selector device_selector;`?

Reply:
It was merged to XMain today.  Hopefully it will show up shortly as a resource.  You can test by adding a -I path and setting up only that file in the right tree.  Might be better to wait, assuming it will get qualified quickly